### PR TITLE
new file added

### PR DESCRIPTION
**Pull request by jinderbrar**
new file added

### CODE REVIEW

File: File1.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+This is the new file for feature3
```
Comment (PiyushSinha-9) [inline File1.txt]: bhai galat hai dursi line likh kuch

Comment (jinderbrar) [reply File1.txt]: ok

**Review by PiyushSinha-9 — APPROVED** [review]

**Review by PiyushSinha-9 — CHANGES_REQUESTED** [review]
#3rd line ho jay

File: File4.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+4444444444444
```
Comment (PiyushSinha-9) [inline File4.txt:1]: 4444444444444 ki jagah kuch formal likh

File: File5.txt [hunk 3]
```diff
@@ -0,0 +1 @@
+5555555555555555555
```
Comment (PiyushSinha-9) [inline File5.txt:1]: sahi hai bhaiii

**Review by PiyushSinha-9 — CHANGES_REQUESTED** [review]
DIY

File: File4.txt (hunk 2 above)
Comment (jinderbrar) [reply File4.txt:1]: ok

File: File5.txt (hunk 3 above)
Comment (jinderbrar) [reply File5.txt:1]: 🆒

**Review by jinderbrar — COMMENTED** [review]
## Changes DOne

**Pull request merged**